react/features/invite/components/dial-in-info-page/DialInInfoPage.web.js

```diff
@@ -190,6 +190,9 @@ class DialInInfoPage extends Component {
      * internal to show the numbers.
      *
      * @param {Object} response - The response from fetching dial-in numbers.
+     * @param {Array|Object} response.numbers - The dial-in numbers.
+     * @param {boolean} reponse.numbersEnabled - Whether or not dial-in is
+     * enabled.
```

Comment:
Sorry to be so picky with this, but if this is about whether the dial-in is enabled, shouldn't we call it dialInEnabled instead of numbersEnabled? At first, I wasn't sure what exactly are we setting here with this numbersEnabled.

Reply:
@aaronkvanmeerten just explained to me that this can't be renamed due to backwards compatibility, so I guess we'll stick with it.